src/Layouts/Partials/Artboard.vala

```diff
@@ -46,24 +46,24 @@ public class Akira.Layouts.Partials.Artboard : Gtk.ListBoxRow {
     public Gtk.ListBox container;
     public int layers_count { get; set; default = 0; }
 
-    public Akira.Models.LayerModel model { get; construct; }
+    public Akira.Lib.Models.CanvasArtboard artboard_model { get; construct; }
 
     private bool _editing { get; set; default = false; }
     public bool editing {
         get { return _editing; } set { _editing = value; }
     }
 
-    public Artboard (Akira.Window window, Akira.Models.LayerModel model) {
+    public Artboard (Akira.Window window, Akira.Lib.Models.CanvasArtboard artboard_model) {
         Object (
             window: window,
-            model: model
+            artboard_model: artboard_model
```

Comment:
Since this is the `model` of the `artboard` and we're only dealing with the `item_model`, it's better and more linear to call this simply `model`

Reply:
Ok, I agree!